src/_settings.scss

```diff
@@ -156,6 +158,14 @@ $bdrs-small: ceil($bdrs-base / 2) !default;
 $bxsh-spread: 3px !default;
 $bxsh-base: 0 1px $bxsh-spread rgba($c-black, .15) !default;
 
+// Layers (z-index) sorting.
+$z-layers: (
+  'modal-basic': 100
+  'cookie-banner': 50,
+  'ad-smartbanner': 10,
+  'card-article-icon': 1,
+  'form-autocompleted-suggests': 1
```

Comment:
This two may or may not collide? I understand you are just simply refactoring out... but its worth setting form-autocompleted-suggest at least to 2?

Reply:
Yes, I would do it, but as you say, it wasn't just a refactor, and it should be tested carefully. Firstly I wanted to set the basis in order to have all `z-index` values sorted, it's much much more than what we have now 😄 .